app/src/main/kotlin/com/intive/picover/common/validator/qualifier/Validator.kt

```diff
@@ -0,0 +1,10 @@
+package com.intive.picover.common.validator.qualifier
+
+import javax.inject.Qualifier
+
+interface Validator {
+
+	@Qualifier
+	@Retention(AnnotationRetention.RUNTIME)
+	annotation class Profile
```

Comment:
I would for creating a strong type (class) instead on using qualifier since it can be avoided in this case but it's rather a personal approach

Reply:
Do you mean to create a class for each validator? Like `ProfileTextValidator(...) : TextValidator(...)`?

Reply:
Exactly, that could be one of the option. 
Apart from that, it shouldn't be named `Profile` since it could suggest that it used for validation of profile fields as a group but it checks only a name, nothing more.